projects/ion/src/lib/table/table.component.scss

```diff
@@ -137,6 +138,7 @@ table {
   justify-content: space-between;
   padding: 16px;
   gap: spacing(2);
+  z-index: 900;
```

Comment:
won't this be a problem when opening a modal?
I suggest creating a pattern for the z-index to solve this problem.

Reply:
That's a great suggestion. Imma work on this right now.

Reply:
Already done!